From 16a7c274989f47b0c0d8ba192a30316b545bd26a Mon Sep 17 00:00:00 2001
From: Wim Taymans <wtaymans@redhat.com>
Date: Fri, 17 Jun 2022 09:02:13 +0200
Subject: [PATCH] settings: remove 44.1KHz from allowed rates again

It seems to cause too many problems. It people want rate switching they
can add an override themselves.
---
 src/daemon/minimal.conf.in  | 2 +-
 src/daemon/pipewire.conf.in | 2 +-
 src/pipewire/settings.c     | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/daemon/minimal.conf.in b/src/daemon/minimal.conf.in
index 49227dddf..8b3ba7640 100644
--- a/src/daemon/minimal.conf.in
+++ b/src/daemon/minimal.conf.in
@@ -27,7 +27,7 @@ context.properties = {
 
     ## Properties for the DSP configuration.
     #default.clock.rate          = 48000
-    #default.clock.allowed-rates = [ 44100 48000 ]
+    #default.clock.allowed-rates = [ 48000 ]
     #default.clock.quantum       = 1024
     #default.clock.min-quantum   = 32
     #default.clock.max-quantum   = 2048
diff --git a/src/daemon/pipewire.conf.in b/src/daemon/pipewire.conf.in
index 787e45f42..3b9f611b7 100644
--- a/src/daemon/pipewire.conf.in
+++ b/src/daemon/pipewire.conf.in
@@ -27,7 +27,7 @@ context.properties = {
 
     ## Properties for the DSP configuration.
     #default.clock.rate          = 48000
-    #default.clock.allowed-rates = [ 44100 48000 ]
+    #default.clock.allowed-rates = [ 48000 ]
     #default.clock.quantum       = 1024
     default.clock.min-quantum   = 16
     #default.clock.max-quantum   = 2048
diff --git a/src/pipewire/settings.c b/src/pipewire/settings.c
index 92d297ca3..c512e9658 100644
--- a/src/pipewire/settings.c
+++ b/src/pipewire/settings.c
@@ -41,7 +41,7 @@
 #define NAME "settings"
 
 #define DEFAULT_CLOCK_RATE			48000u
-#define DEFAULT_CLOCK_RATES			"[ 44100 48000 ]"
+#define DEFAULT_CLOCK_RATES			"[ 48000 ]"
 #define DEFAULT_CLOCK_QUANTUM			1024u
 #define DEFAULT_CLOCK_MIN_QUANTUM		32u
 #define DEFAULT_CLOCK_MAX_QUANTUM		2048u
-- 
GitLab

